doc/design/proposal-chaining-iterative.md

```diff
@@ -282,3 +282,53 @@ const babelOutputToFormat = new Map([
 ]);
 ```
 </details>
+
+## Chaining `readFile` hooks
+
+Say you had a chain of three loaders:
+
+* `zip` adds a virtual filesystem layer for in-zip access
+* `tgz` does the same but for tgz archives
+* `warc` does the same for warc archives.
+
+Following the pattern of `--require`:
+
+```console
+node \
+  --loader zip \
+  --loader tgz \
+  --loader warc
+```
+
+These would be called in the following sequence:
+
+(`zip` OR `defaultReadFile`) → `tgz` → `warc`
+
+1. `defaultReadFile` / `zip` needs to be first to know whether the manifest exists on the actual filesystem, which is fed to the subsequent loader
```

Comment:
I think a brief comment explaining what the manifest is / why it's needed would be helpful for those unfamiliar with the problem-space (which I imagine is most people).

Reply:
Added a section detailing this in the overview document; do you feel it covers enough while not being overly technical?

Reply:
Thanks!